catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -91,6 +101,39 @@ protected SolrCacheMBean getCacheProxy() {
         verify(mbean, times(1)).removeById(idsArray);
     }
 
+    @Test
+    public void testOverMaxItemList() throws Exception {
+        ConsoleOutput consoleOutput = mock(ConsoleOutput.class);
+        int totalResultsToDelete = 100;
+        int resultsPerCFQuery = 22;
+        int numCFCalls = 5;
+        List<String> ids = new ArrayList<>();
+        List<Metacard> mcList = getMetacardList(totalResultsToDelete);
+        for(int i = 0; i < mcList.size(); i++) {
+            ids.add(mcList.get(i).getId());
+        }
+
+        RemoveCommand removeCommand = new RemoveCommand();
+
+        final CatalogFramework catalogFramework = mock(CatalogFramework.class);
+
+        QueryResponse queryResponse = mock(QueryResponse.class);
```

Comment:
Consider placing mock instantiations into a setup method as well as any repeated when statements to clean up this test method. That is the role of the setup. Refer to https://github.com/codice/ddf/pull/2148/files#diff-84f7749e6d7b0fc4c56387a17d2799a8R82 for an example of how to set up class/setup method for better reusability.